docs/managed-datahub/slack/saas-slack-troubleshoot.md

```diff
@@ -69,3 +69,24 @@ If you've subscribed to an entity, and then performed an action (i.e., raised an
 ### There is an issue with DataHub's systems
 If sending a test notification works, and you've verified that none of the above cases apply, then you should contact your DataHub Customer Success rep to help troubleshoot and resolve the issue. 
 
+## Slack bot issues
+This is to troubleshoot issues with your slack bot
+
+### Command failed with error "dispatch_failed"
+If you've installed the slack bot, but your commands are failing with an error 'dispatch_failed', you can try the following to correct it.
+1. Open your acryl instance with the following url: `<your-instance-base-url>/settings/integrations/slack?display_all_configs=true`.
```

Comment:
Should this be DataHub Cloud instance instead?

Reply:
haha good catch